migrations/migrations/1_filters_table.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS filters;
```

Comment:
Is there any sense to use this?

Reply:
This is a part of issue "down migrations"